source/Background/Controller.js

```diff
@@ -7,7 +7,11 @@ import PlugController from '@psychedelic/plug-controller';
 import { validatePrincipalId } from '@shared/utils/ids';
 import { E8S_PER_ICP, CYCLES_PER_TC } from '@shared/constants/currencies';
 import { XTC_FEE } from '@shared/constants/addresses';
-import { getApps, setApps, removeApp } from '@modules';
+import {
+  getApps,
+  setApps,
```

Comment:
We should def have a module for all these things as well instead of having single functions exported

Reply:
Absolutely, I'll include a refactor for the `StorageModule`